Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "gba"
 description = "A crate (and book) for making GBA games with Rust."
-version = "0.3.0"
+version = "0.3.0-pre"
 authors = ["Lokathor <zefria@gmail.com>", "Ketsuban"]
```

Comment:
I use my legal name for git commits and stuff, so `Thomas Winwood <twwinwood@gmail.com>`.